rest/model/codegen.go

```diff
@@ -355,8 +365,8 @@ func generateServiceConversions(structVal *types.Struct, packageName, structName
 		sort.Strings(bfsCode)
 		sort.Strings(tsCode)
 	}
-	if fieldErrs.HasErrors() {
-		return "", fieldErrs.Resolve()
+	if fieldWarnings.HasErrors() {
+		grip.Warning(fieldWarnings.Resolve())
```

Comment:
Sorry I'm missing this context--how/where is Codegen used? Will this log to splunk or the user?